lightning/src/routing/gossip.rs

```diff
@@ -1033,6 +1033,11 @@ pub enum EffectiveCapacity {
 	/// A capacity sufficient to route any payment, typically used for private channels provided by
 	/// an invoice.
 	Infinite,
+	/// The maximum HTLC amount as provided by an invoice route hint.
+	HintMaxHTLC {
```

Comment:
_if_ this is used for blinded paths as well, `Hint` might not be the best name.

Reply:
Ah yeah, there's several places in the API already where we refer to `(BlindedPayInfo, BlindedPath)` things from BOLT12 invoices as "blinded route hints" (and more in #2120). 

That said, I'm not super happy with this name, so v open to alternatives